packages/mobile/src/screens/chat-screen/ChatTextInput.tsx

```diff
@@ -244,7 +246,8 @@ export const ChatTextInput = ({
         style={{
           position: 'relative',
           maxHeight: hasCurrentlyPlayingTrack ? spacing(70) : spacing(80),
-          paddingBottom: Platform.OS === 'ios' ? spacing(1.5) : undefined
+          paddingBottom:
+            Platform.OS === 'ios' ? spacing(1.5) + extraOffset : undefined
```

Comment:
is this not a concern in general for android? or are we waiting to find the right offset?

Reply:
gonna test w android tmr and see if it needs it

Reply:
@amendelsohn see latest commit! tried to leave comments to make it clear